src/index.html

```diff
@@ -8,7 +8,7 @@
           function gtag(){dataLayer.push(arguments);}
           gtag('js', new Date());
 
-          gtag('config', 'UA-84705099-3');
+          gtag('config', 'UA-84705099-3', { 'anonymize_ip': true });
```

Comment:
`ga('set', 'anonymizeIp', true);`

Reply:
See the link in the PR https://developers.google.com/analytics/devguides/collection/gtagjs/ip-anonymization, we're using gtag and this is how they specify it there 👍

Reply:
Got it! Since we don't use other products than Google Analytics we could move to `analytics.js` for the new site!